codalab/bundles/named_bundle.py

```diff
@@ -16,7 +16,7 @@ class NamedBundle(Bundle):
 
     METADATA_SPECS = (
       MetadataSpec('name', basestring, 'Short variable name (not necessarily unique); must conform to %s.' % spec_util.NAME_REGEX.pattern, short_key='n'),
-      MetadataSpec('description', basestring, 'Full description of the bundle.', short_key='d'),
+      MetadataSpec('description', basestring, 'Full description of the bundle.', short_key='m', metavar='MESSAGE'),
```

Comment:
Hmm...isn't this going to break backward compatibility?  I'd rather just disable `-d` for depth because I don't think people are likely to use that, and if they are, then `--depth` is fine.